### PR TITLE
Create Enumerable in Ruby

### PR DESCRIPTION
**Pull request by Atril33**
## **Enumerable**
_**In this Project, I have done a few suggested requirements**_

- Create `my_list.rb` & `my_enumerable.rb` File by using Ruby
- Configure linters and Fixed linters.

### CODE REVIEW

**Review by AlphaNtihinduka — COMMENTED** [review]
Hi @Atril33 ,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
# Invalid Code Review Request ❌ 

You have submitted a project, that is not contributed by all (2) assigned pair group members. Kindly ensure that all assigned members to this project contribute evenly to this task, and then request another code review. In addition, please make your branch name(feature) more descriptive

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready pray

Cheers and Happy coding!clapclapclap

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

File: my_list.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+class MyList
+  include MyEnumerable
+
+  def initialize(*items)
+    @list = items
+  end
+
+  def each
+    @list.each { |ele| yield ele }
+  end
```
Comment (mikethreels) [inline my_list.rb]: - [ ] Please try to avoid using the existing each method but instead loop over it yourself.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- good job creating the methods

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: my_list.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+require_relative 'Enumerable'
+
+class MyList
+  include MyEnumerable
+
+  def initialize(*items)
+    @list = items
+  end
+end
```
Comment (Ismailco) [inline my_list.rb]: - [ ] Kindly make sure to implement the `each` method and as the previous review mentioned consider implementing the `each` functionality on your own and avoid using the existing ruby method `each()`

File: my_enumerable.rb [hunk 3]
```diff
@@ -0,0 +1,17 @@
+module Enumerable
+  def all?
+    each { |item| return false unless yield item }
```
Comment (Ismailco) [inline my_enumerable.rb:3]: - [ ] [OPTIONAL] Kindly make sure to use the `each` method you created to replace the ruby each method in the `all?`, `any?`, and `filter` metods

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @Atril33, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

✔️ No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Felix45 — APPROVED** [review]
# Approved :green_circle: 
Hi @Atril33 & @Ismailco , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :heavy_check_mark: 
- [x] Good code and source file organization :white_check_mark: 
- [x] Enumerable works as expected :white_check_mark: 
- [x] Repository has a professional readme file :white_check_mark:  
- [x] Implemented all the changes from the previous reviewer :white_check_mark:  

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**